guide/ops/upgrade.md

```diff
@@ -0,0 +1,356 @@
+
+2. **Important!** Backup persisted state and custom configuration.
+
+   1. By default, persisted state is located at `/opt/brooklyn/.brooklyn/`..
+      The `persistenceDir` and `persistenceLocation` are configured in the file `./etc/org.apache.brooklyn.osgilauncher.cfg`.
+      The persistence details will be logged in `./log/brooklyn.info.log` at startup time.
+
+   2. Configuration files are in `./etc/`.
+      Any changes to these configuration files will need to be re-applied after reinstalling Brooklyn.
+
+3. Delete the existing Apache Brooklyn install:
+
+   1. Remove Brooklyn package:
+
+          # CentOS / RHEL
+          sudo yum erase apache-brooklyn
+          
+          # Ubuntu / Debian
+          sudo dpkg -r apache-brooklyn
+
+    2. On CentOS 7 run `sudo systemctl daemon-reload`.
+
+    3. Confirm that Brooklyn is definitely not running (see step 1 above).
+
+    4. Delete the Brooklyn install directory: `sudo rm -r /opt/brooklyn`.
+
+4. Make sure you have Java 8.
+   By default CentOS images come with JRE6 which is incompatible version for Brooklyn.
+   If CentOS is prior to 6.8 upgrade nss: `yum -y upgrade nss`
+
+5. Install new version of Apache Brooklyn:
+
+   1. [Download](../misc/download.html) the new RPM/DEB package.
+   
+   2. Install Apache Brooklyn:
+
+          # CentOS / RHEL
+          sudo yum install apache-brooklyn-xxxx.noarch.rpm
+          
+          # Ubuntu / Debian
+          sudo dpkg -i apache-brooklyn-xxxx.all.deb
+
+   3. Wait for Brooklyn to be fully started (i.e. its web-console to be responsive).
+
+6. Restore the persisted state and configuration.
+
+   1. Stop the Brooklyn service:
+
+          # CentOS 7 / RHEL 
+          sudo systemctl stop brooklyn
+          # CentOS 6 and older
+          sudo initctl stop brooklyn
+          
+          # Ubuntu / Debian
+          stop brooklyn
+
+      Confirm that Brooklyn is no longer running (see step 1).
+
+   2. Restore the persisted state directory into `/var/lib/brooklyn` and any changes to the configuration files (see step 2).
+      Ensure owner/permissions are correct for the persisted state directory, e.g.:
+      `sudo chown -r brooklyn:brooklyn /var/lib/brooklyn`
+
+7. Validate that the new release works, by starting in "HOT_BACKUP" mode.
+
+   1. Before starting Brooklyn, reconfigure `./etc/org.apache.brooklyn.osgilauncher.cfg` and set `highAvailabilityMode=HOT_BACKUP`.
+      This way when Brooklyn is started, it will only read and validate the persisted state and will not write into it.
+
+   2. Start Apache Brooklyn:
+
+          # CentOS 7 / RHEL
+          sudo systemctl start brooklyn
+          # CentOS 6 and older
+          sudo initctl start brooklyn
+          
+          # Ubuntu / Debian
+          start brooklyn
+
+   3. Check whether you have rebind ERROR messages in `./log/brooklyn.info.log`, e.g. `sudo grep -E "WARN|ERROR" /opt/brooklyn/log/brooklyn.debug.log`.
```

Comment:
I'd say "ERROR mesages in the brooklyn log, e.g. ..." (rather than the `./log/...`)

Reply:
I'd use `sudo grep -E "Rebind|WARN|ERROR" /var/log/brooklyn/brooklyn.info.log` (important to include something like "Rebind" so the user sees a message about success, as well as some benign (?) warnings).

Reply:
Make sense @aledsage, will update